simapp/simd/cmd/root.go

```diff
@@ -39,12 +42,17 @@ import (
 // main function.
 func NewRootCmd() *cobra.Command {
 	// we "pre"-instantiate the application for getting the injected/configured encoding configuration
-	tempApp := simapp.NewSimApp(log.NewNopLogger(), dbm.NewMemDB(), nil, true, simtestutil.NewAppOptionsWithFlagHome(simapp.DefaultNodeHome))
-	encodingConfig := params.EncodingConfig{
-		InterfaceRegistry: tempApp.InterfaceRegistry(),
-		Codec:             tempApp.AppCodec(),
-		TxConfig:          tempApp.TxConfig(),
-		Amino:             tempApp.LegacyAmino(),
+	encodingConfig := params.EncodingConfig{}
+	var autoCliOpts autocli.AppOptions
```

Comment:
could you quickly explain how this would work with non depinject apps?

Reply:
Moved to app-v1 and app-v2.